core/GlobalState.cc

```diff
@@ -1743,21 +1934,29 @@ unique_ptr<GlobalStateHash> GlobalState::hash() const {
     u4 hierarchyHash = 0;
     UnorderedMap<NameHash, u4> methodHashes;
     int counter = 0;
-    for (const auto &sym : this->symbols) {
-        if (!sym.ignoreInHashing(*this)) {
```

Comment:
seems like this call disappeared. Should we keep it?

Reply:
Oh, I need to re-add this (or inline it into) the other loop so we ignore classes owned by StubModule -- good call.

Reply:
Fixed.